src/tests/ftest/pool/bad_evict.py

```diff
@@ -140,5 +137,4 @@ def test_evict(self):
                         pool.uuid[item] = saveduuid[item]
                 if savedsvc is not None:
                     pool.svc = savedsvc
-                pool.disconnect()
-                pool.destroy(1)
+                pool.destroy(0)
```

Comment:
Given that a large majority of the pool tests were recently moved over to using `TestPool` to enable using dmg to create and destroy the pools, should this change be made here? or is a new PR required? @skirvan44